bottles/frontend/windows/gamescope.py

```diff
@@ -54,6 +58,17 @@ def __init__(self, window, config, **kwargs):
 
         self.__update(config)
 
+        # Set VRR row to visible if the compositor supports VRR
+        if vrr_available:
+            self.row_vrr.set_visible(True)
+
+            if DisplayUtils.display_server_type() == "wayland":
+                warning = _("This feature depends on the version of the compositor.")
+                learn_more = _("Learn more about variable refresh rate")
+                link = f"<a href=\"https://wiki.archlinux.org/title/Variable_refresh_rate#Wayland_configuration\">{learn_more}</a>"
+                message = f"{self.row_vrr.get_subtitle()} {warning} {link}"
+                self.row_vrr.set_subtitle(message)
+
```

Comment:
Let's just cut this out, and say "Enable VRR if available." Far cleaner, doesn't limit us to checks for specific compositors, and doesn't make unnecessary checks.